tests/config/test_data.config

```diff
@@ -310,6 +309,14 @@ params {
                     pangenome_lay                              = "${params.test_data_base}/data/pangenomics/homo_sapiens/odgi/pangenome.lay"
                 }
             }
+            'mskcc' {
```

Comment:
Should we maybe write this in the style of nf-core?

Something roughly like this: 
```
test_data_base_msk = 'https://github.com/mskcc-omics-workflows/test-datasets/raw/'

test_data_msk {
 test_argos_sample_1_tumor_bam = "${params.test_data_base_msk}/bam_files/bams/argos_processed/sample_1_tumor.chr17.rg.md.abra.printreads.bam"

}
```
This might help keep things organized? 

Also there are a few quote and spacing edits in this file that I'm not sure we should merge? I guess they might be from the lint-ing though?

Reply:
Thanks, I fixed it